### PR TITLE
[ADD] New dispatch label ZPL

### PR DESCRIPTION
**Pull request by jcadhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/product/pull/500).

File: product_ux/report/picking_templates.xml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+^FO100,545^FD
+<t t-esc="picking.partner_id.parent_id.mobile"/>
+^FS
+</t>
+</t>
+<t t-else="">
+^FS
+</t>
+<!-- Fin destino -->
+<t t-if="picking.carrier_id">
```
Comment (bruno-zanotti) [inline product_ux/report/picking_templates.xml]: Acá no tendríamos que chequear si existe el campo carrier_id porque lo instala delivery?

File: product_ux/__manifest__.py [hunk 2]
```diff
@@ -32,11 +32,13 @@
         'product',
     ],
     'data': [
+        'report/ir.action.reports.xml',
```
Comment (bruno-zanotti) [inline product_ux/__manifest__.py:35]: Falta modificar la versión del manifest

**Comment by bruno-zanotti** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/product#503

**Comment by fw-bot-adhoc** [comment]
@jcadhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/product#503

**Pull request closed without merging**